README.md

```diff
@@ -27,5 +27,28 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
-****Reasoning for Big Theta Complexity****:
-The divide-and-conquer algorithm for summing an array by dividing it into three parts at each step has a time complexity of Θ(n). This conclusion is based on the analysis using the Master Theorem, where the primary cost comes from the need to solve each of the three sub-problems of size n/3, and the combination step is a simple sum operation that does not significantly add to the computational complexity.
+**Recurrence Relation**:<br />
+In my code, each recursive call divides the array into three parts, which implies that the function is called three times for each division. <br />
+Each call deals with 1/3 of the original problem size <br />
+Lastly, there is a linear amount of work done outside of the recursive calls to calculate the splitting points and then combine the sums. <br />
+
+Therefore we can define: $T(n)$ = $3T(n/3)$ + $f(n)$ <br />
+Solving it: $T(n)$ = $3T(n/3)$ + $cn$ <br />
```

Comment:
The recurrence relation is wrong. There shouldn't be "cn", so nlogn is not a correct answer.